### DISCUSSION TITLE
Claw Back stuck in Limbo for 15+ days

### DISCUSSION THREAD

**Discussion by ChiaFarms** [Chia-Network/chia-blockchain#16028] [Support]
Hello, I clicked on the new Claw Back feature and set it to 20 minutes with my 4 Chia I mined. Now my coins are not at Crypto .com and Not in my Chai wallet. Please someone help. If I click on "Claw back this transaction" I get a window asking me to select a amount or FEE to claw back. I choose every (all 4) amounts to claw back, and I get a error of "No transaction ID is returned". I can send a link to the video if needed.
-=-  Raatt  -=-

**Comment by BrandtH22** [comment]
Hey @ChiaFarms , thank you for reaching out.
As a heads up, we can generally provide more immediate and thorough assistance in our discord server: https://discord.gg/chia

I believe that this issue is related to a known bug in 1.8.2 that causes issues when trying to clawback transactions.
The cause of this bug has been identified and will be resolved in the next release version of chia enabling you to properly clawback the coins.

In order to use this fix today you would need to download and install a very specific version of the chia client and use cli commands in order to do so. I would be more than happy to assist you through this process in our Discord servers support channel (https://discord.gg/chia) or if you do not mind waiting until the next release it will have the ability to properly clawback the coins with the GUI options.

Reference of the bug and its resolution: https://github.com/Chia-Network/chia-blockchain/issues/15703
